.github/workflows/reusable-build-test-release.yml

```diff
@@ -1667,7 +1667,7 @@ jobs:
           node-version: '14'
       - name: Semantic Release
         id: semantic
-        uses: cycjimmy/semantic-release-action@v2.7.0
+        uses: cycjimmy/semantic-release-action@v3.2.0
```

Comment:
We should test it probably with some release with a test repo. It is breaking change and we are passing the admin token there so we can also check what changed in new version

Reply:
btw, GitHub started to support more granular PAT. We should switch to this